deblur/parallel_deblur.py

```diff
@@ -0,0 +1,159 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015, The Deblur Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+from __future__ import division
+import os
+from os.path import join
+from tempfile import mkdtemp
+import multiprocessing as mp
+import traceback
+import sys
+import subprocess
+from functools import partial
+
+
+def deblur_system_call(params, fps):
+    """Build deblur command for subprocess.
+
+    Parameters
+    ----------
+    params: dict
+        parameter settings to pass to deblur
+    fps: tuple
```

Comment:
I think you wrote that code and it was because the input filename and output directory were being stored as a list of tuples [at line 151](https://github.com/ekopylova/deblur/blob/58e5a0a0242fdabdac7458df7417799281bbd07b/deblur/parallel_deblur.py#L151) prior to mapping to functor. Do you want me to change this?